tests/framework/queries.go

```diff
@@ -0,0 +1,310 @@
+package framework
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+func getFirstValueOfVector(query string, promInstance PrometheusInstance) (float64, error) {
```

Comment:
Maybe we just put all of these functions into `prometheus.go`. What do you think?

Reply:
I like this idea so I will do it!